package.json

```diff
@@ -37,7 +39,6 @@
     "@typechain/hardhat": "^6.0.0",
     "@types/chai": "^4.2.21",
     "@types/mocha": "^9.0.0",
-    "@types/node": "^17.0.34",
```

Comment:
node types are not needed anymore?

Reply:
it was giving dependency errors, so I removed it. And that didn't change anything for any operations e.g., tests etc., so apparently it's not needed.